package.json

```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "@fraction/base16-css": "^1.1.0",
-    "@fraction/flotilla": "^4.0.1",
+    "@fraction/flotilla": "github:Powersource/flotilla#add-meme-2",
```

Comment:
todo: use semver again once this is merged

Reply:
just published as 5.2.0 :boom: